CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Breaking
```

Comment:
Let's just call it `Removed`, we've had a section named like this in a couple of previous releases. 

```suggestion
#### Removed
```